bootstrap-salt.sh

```diff
@@ -6538,7 +6541,7 @@ install_opensuse_git_post() {
         [ $fname = "minion" ] && [ "$_INSTALL_MINION" -eq $BS_FALSE ] && continue
         [ $fname = "syndic" ] && [ "$_INSTALL_SYNDIC" -eq $BS_FALSE ] && continue
 
-        if [ -f /bin/systemctl ]; then
+        if [ -f /bin/systemctl ] || [ -f /usr/bin/systemctl ]; then
```

Comment:
Actually, this is probably a nicer alternative that we're using in the `salt-image-builder` now:

```suggestion
       if command -v systemctl; then
```